src/server/helpers/utils.js

```diff
@@ -16,7 +16,7 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
-
+/* eslint valid-jsdoc: "warn" */
```

Comment:
See first comment - in this case, change the rule so that returns aren't required, since that seems fairly sensible.